ToolBox/SECDataDownloader/SECDataConverter.cs

```diff
@@ -100,7 +93,7 @@ public SECDataConverter(string rawSource, string destination, string knownEquity
                         symbol.Add(tickerCik[0]);
                     }
                 );
-            
+
```

Comment:
Unrelated to this PR, but this constructor does _a lot_ of work. Further, it uses a pattern I would try to stay away from where we use the `ToList()` LINQ method simply so we can use the `ForEach` method instead of using `for each`. The difference is bigger than you may expect. The `ToList()` method will obviously require complete enumeration of the underlying enumerable along with the requisite allocations. Give the `rankAndFileData` may not be large, this may not be critical. In addition, since we're reading from disk, we can enumerate the lines one by one instead of using `Split('\n')` -- _further_ we're not guaranteed that splitting on `\n` characters is sufficient. Windows will use CRLF `\n\r` as line break, so all of these lines would have  an initial `\r` character.

Instead, it's better practice to stream the file line-by-line using something like this:
``` csharp
foreach (var line in File.ReadLines(Path.Combine(RawSource, "cik-ticker-mappings-rankandfile.txt"))
{
    var tickerInfo = line.Split('|');
    // process  as before
}
```

This removes a couple of issues, the biggest of which is using `string.Split` on an entire file's worth of data. The `Split` function is notoriously memory inefficient as well as potentially _incorrect_ (windows v linux), especially when streaming it from disk using `File.ReadLines` is simpler, more efficient, and _arguably_ more readable. This also removes the unnecessary allocation of the intermediate list just to make the `ForEach` method available _and also_ completely removes the need for putting the entire file in memory.